packages/react-native-gesture-handler/android/src/main/java/com/swmansion/gesturehandler/core/GestureHandler.kt

```diff
@@ -863,7 +864,7 @@ open class GestureHandler {
   abstract class Factory<T : GestureHandler> {
     abstract val type: Class<T>
     abstract val name: String
-
+    private val interactionManager = RNGestureHandlerInteractionManager()
```

Comment:
This cannot be local to a single gesture, as it's responsible for deciding whether a gesture should fail or continue. It needs information about relations for all currently attached gestures.